codeface_extraction/codeface_extraction.py

```diff
@@ -92,6 +92,8 @@ def get_parser():
                             default='codeface.conf')
     run_parser.add_argument('-p', '--project', help="Project configuration file",
                             required=True)
+    run_parser.add_argument('-i', '--implementation', help="Enable extraction of the source code of functions",
+                            type=lambda x: (str(x).lower() == 'true'), default=False)
```

Comment:
Please use `action='store_true'` (see [here](https://docs.python.org/2.7/library/argparse.html#action) for more details).